dataland-frontend/src/utils/Debounce.ts

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This is used only for applySearchFilter in DatasetOverviewTable. Can we talk about what it exactly does and why we need it?

Reply:
I am also interested in understanding this :)